pom.xml

```diff
@@ -49,6 +49,7 @@
     <spotless.check.skip>false</spotless.check.skip>
     <spotbugs.effort>Max</spotbugs.effort>
     <configuration-as-code.version>1836.vccda_4a_122a_a_e</configuration-as-code.version>
+    <hpi.compatibleSinceVersion>4.431</hpi.compatibleSinceVersion>
```

Comment:
config is migrated, but the Jenkins config-as-code format is no longer compatable, so noting so users get a warning before updating.